data/json/items/tool_armor.json

```
@@ -2026,23 +2026,12 @@
     "type": "TOOL_ARMOR",
     "category": "armor",
     "name": { "str": "survivor firemask" },
-    "description": "A custom-built, Nomex-insulated gas mask that covers the face and eyes.  It provides excellent protection from heat, smoke, tear gas, and shrapnel.  It must be prepared before use.",
-    "weight": "708 g",
+    "description": "A custom-made leather gasmask faced with nomex.  It must be prepared before use.",
```

```suggestion
    "description": "A custom-made leather gasmask faced with Nomex.  It must be prepared before use.",
```
*Angry DuPont noises*